src/SmartComponents/SystemDetail/SystemDetail.js

```diff
@@ -8,12 +8,13 @@ import { intl } from '../../Utilities/IntlProvider';
 import messages from '../../Messages';
 import { NoSystemData } from '../../PresentationalComponents/Snippets/NoSystemData';
 import propTypes from 'prop-types';
+import { isSet } from '../../Utilities/Helpers';
 
 const SystemDetail = ({ isInventoryApp }) => {
     const [activeTabKey, setActiveTabKey] = React.useState(0);
     const [areTabsHidden, setTabsHidden] = React.useState(false);
 
-    const entity = useSelector(({ entityDetails }) => entityDetails.entity && entityDetails.entity || {});
+    const entity = useSelector(({ entityDetails }) => isSet(entityDetails.entity) && entityDetails.entity || {});
```

Comment:
```suggestion
    const entity = useSelector(({ entityDetails }) => entityDetails.entity || {});
```
Why so complicated? It should work the same